src/submission/upload/ValidationProgress.jsx

```diff
@@ -1,7 +1,15 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
 import ProgressText from './ProgressText.jsx'
-import * as STATUS from '../../constants/statusCodes.js'
+import {
+  PARSED_WITH_ERRORS,
+  VALIDATING,
+  UPLOADING
+} from '../../constants/statusCodes.js'
+/* TODO
```

Comment:
Just a reminder that we should watch what happens here in certain cases, like a clean file.